docs/gallery-examples/example7_alma.py

```diff
@@ -10,6 +10,7 @@
 from astropy.io import fits
 from astropy import wcs
 from astropy import log
+from astropy.logger import log
```

Comment:
keep this example as is. We need to decide the concept about these examples, probably it would be better to make them part of the narrative docs, but that's beyond the scope of this PR.